pyqt/pyqt.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "pyqt",
+    "config-opts": [
+        "--concatenate=1",
+        "--confirm-license",
+        "--debug",
+        "--no-designer-plugin",
+        "--no-docstrings",
+        "--no-make",
+        "--no-qml-plugin",
+        "--no-tools",
+        "--verbose",
+        "--api-dir=/app/share/qt/qsci/api/python",
+        "--qmake-setting=QMAKE_CFLAGS_RELEASE='-I/usr/include/python_pyver/'",
+        "--qmake-setting=QMAKE_CXXFLAGS_RELEASE='-I/usr/include/python_pyver/'",
+        "--target-dir=/app/lib/python_pyver/site-packages",
+        "--qt-shared"
+    ],
+    "make-args": [
+        "--directory=build"
+    ],
+    "make-install-args": [
+        "--directory=build"
+    ],
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://files.pythonhosted.org/packages/3b/27/fd81188a35f37be9b3b4c2db1654d9439d1418823916fe702ac3658c9c41/PyQt5-5.15.6.tar.gz",
+            "sha256": "80343bcab95ffba619f2ed2467fd828ffeb0a251ad7225be5fc06dcc333af452",
+            "x-checker-data": {
+                "type": "pypi",
+                "name": "PyQt5"
+            }
+        },
+        {
+            "type": "file",
+            "path": "configure"
+        },
+        {
+            "type": "file",
+            "path": "../sip/sip-build-configure.in",
+            "dest-filename": "configure.sip-build"
+        },
+        {
+            "type": "shell",
+            "commands": [
+                "echo '[tool.sip.bindings.QtGui]' >> pyproject.toml",
+                "echo 'disabled-features = [\"PyQt_Desktop_OpenGL\"]' >> pyproject.toml"
+            ],
+            "only-arches": [
+                "aarch64"
+            ]
+        }
```

Comment:
Better to use `build-options` to specific `env` or `config-opts`

Reply:
The `sip-build` will accept `--disabled-feature PyQt_Desktop_OpenGL`

Reply:
Right! I'll change this. Thanks.